installation/install-by-msi.md

```diff
@@ -67,29 +64,36 @@ Open `C:/opt/td-agent/etc/td-agent/td-agent.conf` with a text editor. Replace th
 </match>
 ```
 
-### Step 3: Launch Td-agent Command Prompt
+### Step 3: Launch Fluent Package Command Prompt with Administrator privilege
 
-Open Windows Start menu, and search `Td-agent Command Prompt`. In most environments, the program will be found right under the "Recently Added" section.
+Open Windows Start menu, and search `Fluent Package Command Prompt`. In most environments, the program will be found right under the "Recently Added" section or "Best match" section.
 
-![Windows start menu and Td-agent Command Prompt](../.gitbook/assets/td-agent4-menu.png)
+![Windows start menu and Fluent Package Command Prompt](../.gitbook/assets/fluent-package5-menu.png)
 
-`Td-agent Command Prompt` is basically `cmd.exe`, with a few PATH tweaks for `td-agent` programs. Use this program whenever you need to interact with `td-agent`.
+`Fluent Package Command Prompt` is basically `cmd.exe`, with a few PATH tweaks for Fluentd programs. Use this program whenever you need to interact with Fluentd.
 
-### Step 4: Run `td-agent`
+### Step 4: Run `fluentd`
 
-Type the following command into `Td-agent Command Prompt`:
+Type the following command into `Fluent Package Command Prompt` with Administrator privilege:
 
 ```text
-C:\opt\td-agent> td-agent
+C:\opt\fluent> fluentd
 ```
 
-Now `td-agent` starts listening to Windows Eventlog, and will print records to stdout as they occur.
+Now `fluentd` starts listening to Windows Eventlog, and will print records to stdout as they occur.
+
+![Fluent Package Command Prompt](../.gitbook/assets/fluent-package5-prompt.png)
+
+### Step 5: Run `fluentd` as Windows service
 
-![Td-agent Command Prompt](../.gitbook/assets/td-agent4-prompt.png)
+Since version 5.0.0, `fluentd` is registered as a Windows service permanently by the msi installer,
+but not enabled by default. You must start `fluentd` service manually.
```

Comment:
Since the first half is about v4, how about summarizing it as follows?

```suggestion
Fluentd is registered as a Windows service permanently by the msi installer.
Since version 5.0.0, the service does not automatically start after installed. You must manually start it.
```

Reply:
Applied, thanks!